runtime/shared_common/CacheMap.cpp

```diff
@@ -4585,7 +4592,7 @@ SH_CacheMap::printAllCacheStats(J9VMThread* currentThread, UDATA showFlags, SH_C
 						}
 					}
 				} else if (J9SHR_DATA_TYPE_STARTUP_HINTS == type) {
-					if ((PRINTSTATS_SHOW_BYTEDATA == (showFlags & PRINTSTATS_SHOW_BYTEDATA))
+					if ((J9_ARE_ANY_BITS_SET(showFlags, PRINTSTATS_SHOW_BYTEDATA | PRINTSTATS_SHOW_STARTUPHINT))
```

Comment:
Is PRINTSTATS_SHOW_BYTEDATA needed since startuphint is a separate category? Nothing else seems to do this.

Reply:
It is not needed, removed.